teuthology/orchestra/console.py

```diff
@@ -35,6 +35,18 @@ def __init__(self, name, ipmiuser, ipmipass, ipmidomain, logfile=None,
         self.has_credentials = all(
             [self.ipmiuser, self.ipmipass, self.ipmidomain]
         )
+        self.conserver_master = config.conserver_master
+        self.conserver_port = config.conserver_port
+        conserver_client_found = subprocess.Popen(
+            'which console',
```

Comment:
Mech: yeah let's discuss in the mroning.  which: OK, but we should specify absolute path.